tools/health/isAlive

```diff
@@ -85,13 +85,6 @@ def isElkAlive() :
     okString = '"cluster_name"'
     return monitorUtil.outputChecker(cmd, okString, 1)
 
-def isKafkaAlive() :
```

Comment:
i've discussed this with @markusthoemmes - while it appears we don't use this method anymore, it does make `isAlive` inconsistent in that all the other components are known to this script.